specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -2478,10 +2490,22 @@
           "description": "Provisioning status of the profile.",
           "type": "string"
         },
-        "frontdoorId": {
+        "frontDoorId": {
```

Comment:
I think it is not recommended for change of property name.

Reply:
We needed to make this change to keep consistency with the casing returned by our service. Additionally, this is a new api version which will not impact previous versions of the api.